apitools/base/py/transfer.py

```diff
@@ -378,7 +386,8 @@ def __ProcessResponse(self, response):
             self.stream.write('')
         return response
 
-    def GetRange(self, start, end=None, additional_headers=None):
+    def GetRange(self, start, end=None, additional_headers=None,
+                 ignore_chunksize=False):
```

Comment:
This will require a gsutil revision (and thus an apitools release), which is fine.